docs/Account sign up - Approved and rejected sign up flows.md

```diff
@@ -0,0 +1,148 @@
+# Microsoft Dynamics 365 Fraud Protection - API examples
+## Account sign up - Approved and rejected sign up flows
+
+After sending Dynamics 365 Fraud Protection a sign up event, it is up to you to use the Dynamics 365 Fraud Protection risk score to either continue with or stop your sign up workflow. Inform Dynamics 365 Fraud Protection of the final sign up status to improve Dynamics 365 Fraud Protection models.
+
+**Important:** Dynamics 365 Fraud Protection does not yet support rules for sign up assessment like it does for purchase assessment. Implement rules in your application to handle the sign up response risk score. For instance, you may decide that any score higher than some threshold is too risky and you will reject those sign ups. Once Dynamics 365 Fraud Protection supports sign up rules, configure those rules directly in Dynamics 365 Fraud Protection so that your applications can simply honor the merchant rule decision in the sign up response. This sample site randomly rejects 1 out of 3 sign ups to demonstrate the concept.
+
+## Helpful links
+- [Calling Dynamics 365 Fraud Protection](./Authenticate&#32;and&#32;call&#32;Fraud&#32;Protection.md)
+- [SignUp - Data model and endpoint](https://apidocs.microsoft.com/services/dynamics365fraudprotection#/v1.0/V1.0MerchantservicesEventsSignUpPost)
+- [SignUp Status - Data model and endpoint](https://apidocs.microsoft.com/services/dynamics365fraudprotection#/v1.0/V1.0MerchantservicesEventsSignUpStatusPost)
+- [Sample site - Register a user](../src/Web/Controllers/AccountController.cs) (see Register POST method)
+- [Sample site - Dynamics 365 Fraud Protection service](../src/Infrastructure/Services/FraudProtectionService.cs) (see PostSignUp and PostSignUpStatus methods)
+
+## Sign up flow
+The sign up event flow consists of:
+1. Sign up event
+1. Sign up status event
+
+**NOTES**
+- You are not required to send these events, much less send them in any particular order, or within any specific time. The sample site does, in fact, follow this order. 
+- As the merchant, you decide when to call the SignUp API. It may be before or after you have created an account in your database. The sample site calls the SignUp API before it creates a user in its database and only creates the user in the database if it decides to approve the sign up. Notice how this sample site does not send a user ID until the sign up status event, and even then, only when it decides to approve the user. This implementation decision is guided by the sample site's behavior.
+- Set the 'assessmentType' field based on if you plan to use the Dynamics 365 Fraud Protection risk score:
+  - Pass 'Evaluate' if you do NOT plan to use the risk score and are still evaluating Dynamics 365 Fraud Protection against your existing fraud solution.
```

Comment:
Do other apis follow this evaluate/protect assessmentType pattern? I don't recall seeing it much.